django_codemod/cli.py

```diff
@@ -117,9 +118,17 @@ def djcodemod():
     type=VersionParamType(DEPRECATED_IN),
     multiple=True,
 )
+@click.option(
+    "--codemod",
+    "codemod",
+    help="Choose a specific codemod to run. Can be repeated.",
+    type=click.Choice(BY_NAME.keys()),
```

Comment:
> This does make the output of `--help` a little awkward

Is it caused by this line? I think it would be more used friendly to mention the `list` subcommand to see what can be specified...

We would need to validate the name ourselves, but it should be relatively simple to do no?

Reply:
Yeah. Though you know what, I'll just [subclass `click.Choice` and override this bit...](https://github.com/pallets/click/blob/801a29f1a85db0845d1277d8e031800b003eff61/src/click/types.py#L219-L220)

Reply:
Nice idea!